pages/jobs.mdx

```diff
@@ -32,7 +32,8 @@ We offer student assistant positions to work with us on research and teaching pr
 - **C++ Developer Position**: job description and more information [here](/cpp-student-job).
 - **Java / JavaScript Developer Position**: For more details and application contact [Mohamed Sherif](/MohamedAhmedSherif)
 - **Python Developer Position**: For more details and application contact [Stefan Heindorf](/StefanHeindorf)
-
+- **Java Developer Position**: job description and more information [here](/hiring/tebaqa-student-job)
+- **Python (ML/NLP) Developer Position**: job description and more information [here](/hiring/porque-student-job)
```

Comment:
The links that you defined here, point to the files but the files themselves state that their path should be under `/jobs/`. Did you test the page? I am not sure whether it will work like that :thinking:

Reply:
I did test it locally beforehand, it seems to work fine. You are right about the '/jobs' part in path but somehow that one doesn't seem to work 😅